apps/web/src/services/dashboards/store/dashboard-detail-info.ts

```diff
@@ -82,21 +85,32 @@ const DASHBOARD_DEFAULT = Object.freeze<{ settings: DashboardSettings }>({
 });
 
 const refineVariablesSchema = (variablesSchemaInfo?: DashboardVariablesSchema, labels?: string[]): DashboardVariablesSchema => {
-    if (labels?.includes('Asset')) {
-        return {
-            properties: { ...assetManagedDashboardVariablesSchema.properties, ...variablesSchemaInfo?.properties ?? {} },
-            order: union(assetManagedDashboardVariablesSchema.order, variablesSchemaInfo?.order ?? []),
-        };
+    if (isEmpty(variablesSchemaInfo?.properties)) { // create dashboard case
+        const _managedDashboardVariablesSchema = cloneDeep(managedDashboardVariablesSchema);
+        if (labels?.includes('Asset')) {
+            Object.entries(_managedDashboardVariablesSchema.properties).forEach(([key, value]) => {
+                if (Object.keys(ASSET_REFERENCE_TYPE_INFO).includes(key)) {
```

Comment:
suggestion: I recommend to use javascript `Map` to browse `managedDashboardVariablesSchema.properties`.